src/Directory.Build.props

```diff
@@ -31,7 +31,7 @@
     <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
 
-    <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <NoWarn>$(NoWarn);CS1591;CA1401</NoWarn>
```

Comment:
How very strange that we got away this long with that NoWarn setting. Do you have an extra VS extension installed or something that led to you seeing these warnings?

Reply:
Maybe during my attempts making the Roslyn analyzers auto-fix I installed [this](https://docs.microsoft.com/en-us/visualstudio/code-quality/install-fxcop-analyzers?view=vs-2019#to-install-fxcop-analyzers-as-a-vsix).